plugins/regmap/src/deviceregistermap.cpp

```diff
@@ -132,16 +132,29 @@ DeviceRegisterMap::DeviceRegisterMap(RegisterMapTemplate *registerMapTemplate, R
 	}
 
 	QObject::connect(registerController, &RegisterController::requestRead, registerMapValues,
-			 &RegisterMapValues::requestRead);
+			 [=](uint32_t address) {
+				 if(isAxi) {
```

Comment:
Why do you need to propagate isAxi thorugh the code. You could simply create a new R/W strategy if needed.